Makefile

```diff
@@ -212,6 +212,10 @@ MOCKGEN = $(shell pwd)/bin/mockgen
 mockgen:
 	$(call go-get-tool,$(MOCKGEN),github.com/golang/mock/mockgen@latest)
 
+OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
+operator-sdk:
+	$(call curl-get-tool,$(OPERATOR_SDK),https://github.com/operator-framework/operator-sdk/releases/download/v1.7.1/operator-sdk_linux_amd64)
```

Comment:
This downloads only the linux version.

Can we add a `[[ $OSTYPE == 'darwin'* ]]` check, so we can download the OSX one

Reply:
Good idea

Reply:
@Mojachieee Could you please pick this up? I suggest to replace `linux` to some `$(uname)` magic, but i don`t have macos, so can`t verify the change.

Reply:
This works on linux: `$(uname  | tr '[:upper:]' '[:lower:]')`